### DISCUSSION TITLE
Can I use code-agent on a frontend repo?

### DISCUSSION THREAD

**Discussion by rohit-gulati-fourkites** [Codium-ai/cover-agent#188] [General]
I want to create unit test cases for all the .tsx files in my repo to improve the unit test coverage

**Comment by EmbeddedDevops1** [comment]
Honestly, as long as you give enough context to the LLM it should be able to do whatever you want. The issue is that it may not know the structure of your TSX file. My best recommendation is to manually create a prompt and test it with an LLM. Based on that you can tweak Cover Agent to meet your needs.
